test-cypress/stubs/ttyg/ttyg-stubs.js

```diff
@@ -1,4 +1,5 @@
 import {Stubs} from "../stubs";
+import {CHAT_MESSAGE_ROLE} from "../../../src/js/angular/models/ttyg/chat-message";
```

Comment:
No, don't do that. Define constants in the test package and don't depend on the src

Reply:
Removed